packages/opentelemetry-types/src/distributed_context/EntryValue.ts

```diff
@@ -19,7 +19,12 @@
  * DistributedContext}.
  */
 export interface EntryValue {
+  /** `String` value of the `EntryValue`. */
   value: string;
+  /**
+   * `EntryTtl` is an integer that represents number of hops an entry can
+   * propagate.
+   */
   ttl?: EntryTtl;
```

Comment:
should this say "`ttl` is an integer..."?

Reply:
Agreed, done.